build.sbt

```diff
@@ -10,7 +10,7 @@ libraryDependencies ++= Seq(
   jdbc,
   evolutions,
   "com.adrianhurt" %% "play-bootstrap" % "1.0-P25-B3",
-  "com.typesafe.play" %% "anorm" % "2.5.1",
+  "com.typesafe.play" %% "anorm" % "2.5.0",
```

Comment:
It's fixed with Anorm master, unfortunately, the fix was not backport on the 2.5.x branch: https://github.com/playframework/anorm/pull/109